ci/travis/install-dependencies.sh

```diff
@@ -58,7 +58,7 @@ elif [[ "$LINT" == "1" ]]; then
   export PATH="$HOME/miniconda/bin:$PATH"
   # Install Python linting tools.
   pip install -q flake8 flake8-comprehensions
-  pip install flake8-quotes==1.0.0
+  pip install flake8-quotes
```

Comment:
I think we should use the fixed version here to avoid any instability caused by the release of `flake8-quote`:
`pip install flake8-quotes==2.0.0`